docs/index.rst

```diff
@@ -123,17 +124,20 @@ ar71xx-generic
   - TL-WDR4300 (v1)
   - TL-WR1043N/ND (v1, v2)
   - TL-WR703N (v1)
-  - TL-WR710N (v1)
+  - TL-WR710N (v1, v2)
   - TL-WR740N (v1, v3, v4, v5)
-  - TL-WR741N/ND (v1, v2, v4, v5)
+  - TL-WR741ND (v1, v2, v4, v5)
   - TL-WR743N/ND (v1, v2)
   - TL-WR841N/ND (v3, v5, v7, v8, v9, v10)
   - TL-WR842N/ND (v1, v2)
-  - TL-WR941N/ND (v2, v3, v4, v5)
+  - TL-WR901ND (v1)
+  - TL-WR940N (v1, v2, v3)
```

Comment:
v1 and v2 are still marked BROKEN because they are untested, so they should not appear in the support list.